src/particle/__init__.py

```diff
@@ -32,6 +32,7 @@
 __all__ = (
     "Charge",
     "Geant3ID",
+    "Corsika7ID",
```

Comment:
Content-free comment but can you add this item in alphabetic order, hence one line above? I believe the CI checks we have in fact would stumble over this, but ...